src/main/java/fi/fmi/avi/model/immutable/PhenomenonGeometryImpl.java

```diff
@@ -9,9 +9,9 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
+import fi.fmi.avi.model.PhenomenonGeometry;
 import fi.fmi.avi.model.TacOrGeoGeometry;
-import fi.fmi.avi.model.immutable.TacOrGeoGeometryImpl;
-import fi.fmi.avi.model.sigmet.PhenomenonGeometry;
+import fi.fmi.avi.model.immutable.PhenomenonGeometryImpl_Builder;
```

Comment:
Redundant import. This class could probably be cleaned formatted by FMI IDEA rules.